CMakeLists.txt

```diff
@@ -23,11 +28,19 @@ add_executable(avro2json
   src/avro2json.c
   src/logical.c)
 
+if (WIN32)
+  set(ADDITIONAL_INCLUDE_DIRS include/windows;${VCPKG_INSTALLED_DIR}/x64-windows-release/include/jemalloc)
+else (WIN32)
+  set(ADDITIONAL_INCLUDE_DIRS)
+endif (WIN32)
+
 target_include_directories(avro2json
-  PRIVATE ${AVRO_INCLUDE_DIR})
+  PRIVATE ${AVRO_INCLUDE_DIR}
+  PRIVATE ${ADDITIONAL_INCLUDE_DIRS})
```

Comment:
Probably worth moving the closing ")" to a new line, so that the next time you add something, you won't need to edit this line. Will make using `git blame` easier :)